games/main.tex

```diff
@@ -64,47 +64,36 @@
-in their mass adoption to real-world problems. Transaction costs for micropayments,
-speed and volume limitations are likely problematic even for evolved blockchains
-implementing sharding and proof of stake consensus.
-
-Side chains and state channels have been proposed to remedy these shortcomings.
-While inspired by these approaches, in this paper we take a different approach.
-The framework to support incentive systems for decentralised services
-was first conceived of in the context of designing storage incentives
-for a decentralised content delivery network.
-
-Peers engage in local exchange of data, promises, requests, deliveries (swap).
-Nodes organise in a network, use relaying
-to extend the scope of local interactions, and register with a deposit
-(swear) to offer service guarantees enforced by a courtroom contract
-making them accountable in case of non-compliance (swindle).
+Public proof-of-resource blockchains implement a decentralised consensus mechanism. Consensus is reached on the validity and ordering of
+transactions in a state machine. State transitions are triggered by transactions, and clients run a virtual machine to calculate these state transitions. In this process, transactions refer to specific accounts which store programs - called ``smart contracts'' - and the clients trigger functions of these programs to calculate the state transition. Limited only by the expressive power of these smart contracts, the blockchain can govern a myriad of rules of interaction, and enforce agreements.
+
+In the context of data storage and provision, we can imagine carefully designed economic incentive systems that are driven by transparent smart contracts, which regulate the peer-tp-peer data flow in such a way that the emergent properties of the network are beneficial to all of its users.
+Such a system can act as a fully decentralised application platform, providing useful services while potentially
+disintermediating trusted third parties on which such services had to rely in the past.
+
+While the Turing-complete blockchain is sometimes refered to as ``the final missing piece'' of the puzzle to bring the cypherpunk vision
+of decentralisation to its completion, problems with their scalability are still a major bottleneck preventing mass adoption to real-world problems. These scalability issues include the high transaction costs for micropayments, the inherent speed and volume limitations in current blockchain designs, and are likely to remain problematic even for blockchains implementing more avanced technologies such as ``sharding'' and ``proof of stake''.
+
+In recent years, ``Side chains'' and ``state channels'' have been proposed as technologies that could remedy these shortcomings.
+While inspired by these approaches, in this paper we take a different approach. %no we don't. we're really just describing one version of a state channel, but who cares, right?
```

Comment:
>  %no we don't. we're really just describing one version of a state channel, but who cares, right?

probably, then let's take a definition and apply it

Reply:
How about
```
In recent years, ``Side chains'' and ``state channels'' have been proposed as technologies that could remedy these shortcomings, and they directly inspired some of the approaches taken in this paper.
```